staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -256,27 +297,25 @@ func (cfgCtl *configController) processNextWorkItem() bool {
 	return true
 }
 
-// syncOne attempts to sync all the API Priority and Fairness config
-// objects.  It either succeeds and returns `true` or logs an error
-// and returns `false`.
-func (cfgCtl *configController) syncOne() bool {
+func (cfgCtl *configController) SyncOne() (bool, bool, bool, time.Duration) {
+	klog.V(5).Infof("%s SyncOne at %s", cfgCtl.name, cfgCtl.clock.Now().Format(timeFmt))
 	all := labels.Everything()
 	newPLs, err := cfgCtl.plLister.List(all)
 	if err != nil {
 		klog.Errorf("Unable to list PriorityLevelConfiguration objects: %s", err.Error())
-		return false
+		return false, false, true, 0
 	}
 	newFSs, err := cfgCtl.fsLister.List(all)
 	if err != nil {
 		klog.Errorf("Unable to list FlowSchema objects: %s", err.Error())
-		return false
+		return false, false, true, 0
 	}
-	err = cfgCtl.digestConfigObjects(newPLs, newFSs)
+	triedWrites, didWrites, wait, err := cfgCtl.digestConfigObjects(newPLs, newFSs)
 	if err == nil {
-		return true
+		return triedWrites, didWrites, false, wait
 	}
 	klog.Error(err)
-	return false
+	return triedWrites, didWrites, true, 0
```

Comment:
I don't  see usage of the triedWrites or didWrites in the product code.  We normally check writes in unit tests by inspecting the .Actions of the fake clients.

Reply:
These are for an integration test; the clients are not fakes.